tls_credentials.h is not found when building ES-Wifi with CONFIG_TLS_CREDENTIALS (v3.7 branch)

### Describe the bug

On v3.7-branch, tls_credentials.h is not found when building ES-Wifi with CONFIG_TLS_CREDENTIALS.
There is a missing include path in the ES-Wifi driver CMakeLists.txt file.

### Regression

- [ ] This is a regression.

### Steps to reproduce

Build a wifi sample with the b-l4s5I-iot01a and CONFIG_TLS_CREDENTIALS set.

### Relevant log output

```shell
/home/joel/Projects/Mender/workspace/zephyr/drivers/wifi/eswifi/eswifi_socket_offload.c:22:10: fatal error: tls_internal.h: No such file or directory
   22 | #include "tls_internal.h"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
```

### Impact

Not sure

### Environment

v3.7-branch

### Additional Context

_No response_

--- a/drivers/wifi/eswifi/CMakeLists.txt
+++ b/drivers/wifi/eswifi/CMakeLists.txt
@@ -8,6 +8,10 @@ if(CONFIG_WIFI_ESWIFI)
   ${ZEPHYR_BASE}/subsys/net/lib/sockets
   )
 
+  zephyr_library_include_directories_ifdef(CONFIG_TLS_CREDENTIALS
+  ${ZEPHYR_BASE}/subsys/net/lib/tls_credentials
+  )
+
   zephyr_library_sources(
     eswifi_core.c
     eswifi_offload.c

